lib/nimble_template/helpers/github.ex

```diff
@@ -9,4 +9,10 @@ defmodule NimbleTemplate.GithubHelper do
 
   def generate_github_action?(),
     do: Mix.shell().yes?("\nDo you want to generate the Github Action workflow?")
+
+  def generate_github_wiki?(),
+    do:
+      Mix.shell().yes?(
+        "\nDo you want to publish a Github Wiki for this project? You'd need to manually create the first Github Wiki Page and set the GH_TOKEN and GH_EMAIL secret for this properly function."
```

Comment:
Not sure, should it be

```suggestion
        "\nDo you want to publish a Github Wiki for this project? You'd need to manually create the first Github Wiki Page and set the GH_TOKEN and GH_EMAIL secret for this to properly function."
```